lib/pharos/scripts/configure-cri-o.sh

```diff
@@ -14,3 +14,14 @@ systemctl enable crio
 # remove unnecessary cni plugins
 rm /etc/cni/net.d/100-crio-bridge.conf /etc/cni/net.d/200-loopback.conf || true
 systemctl start crio
+
+# Install crictl binary if needed
+CRICTL_DOWNLOAD_SHA="597a4db0289870d81d0377396ddaf4c23725a47b33b30856e6291c2b958786f3"
+
+if ! which crictl > /dev/null || [ "$(crictl -v)" != "$CRICTL_VERSION" ]; then
+    # Not installed or wrong version
+    curl -sSL https://bintray.com/kontena/pharos-bin/download_file?file_path=crictl-${CRICTL_VERSION}-linux-${CPU_ARCH}.tar.gz -o /tmp/crictl.tar.gz
+    echo "$CRICTL_DOWNLOAD_SHA  /tmp/crictl.tar.gz" | shasum -a256 -c
```

Comment:
I'm confused, wasn't this supposed to be using a signed `.deb`? The bintray repo has `.asc` signatures for the `.tar.gz' files, but this isn't checking those?

Reply:
NVM, I misread the earlier discussion... this is fine with a SHA256SUM, although a custom `.deb` in a signed APT repo would be ideal.

Reply:
I'll make it use the .asc for verification

Reply:
done